recipes/jinja2cpp/1.1.0/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 2.8.0)
+project(Jinja2CppTestPackage)
+
+set(CMAKE_CXX_STANDARD 14)
+
+find_package(jinja2cpp REQUIRED)
+
+if (${CMAKE_CXX_COMPILER_ID} MATCHES "Clang" OR ${CMAKE_CXX_COMPILER_ID} MATCHES "GNU")
```

Comment:
You can do `if (CMAKE_CXX_COMPILER_ID MATCHES MSVC)`, I think.
See https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_COMPILER_ID.html

Reply:
Yes. Sure.